webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -16,15 +17,26 @@ const Breadcrumb = ({
 
   return (
     <PfBreadcrumb title={title} {...props}>
-      {items.map((item, index) => (
-        <PfBreadcrumb.Item
-          key={index}
-          active={index === items.length - 1}
-          onClick={item.onClick}
-          href={item.url}
-          dangerouslySetInnerHTML={{ __html: item.caption }}
-        />
-      ))}
+      {items.map((item, index) => {
+        const active = index === items.length - 1;
+        const { caption, caption: { icon, text } } = item;
+
+        return (
+          <PfBreadcrumb.Item
+            key={index}
+            active={active}
+            onClick={item.onClick}
+            href={item.url}
+            title={item.caption.text || item.caption}
+          >
+            {icon && <img src={icon} />}
+            {' '}
+            {text || caption}
+            {' '}
+            { active && <span id='hostFQDN' > {titleAddition} </span>}
```

Comment:
'hostFQDN' id seems too resource specific for this generic component.

I didn't do any testing, but besides that the design looks fine to me.

Reply:
you're right, I chose it because the integration test which catches this id. I'll change the id to something more generic here and in the test.